site/content/en/faq.md

```diff
@@ -388,46 +253,334 @@ information, or simply redirect the reader to your `LICENSES/` directory.
+SPDX-License-Identifier = "CC0-1.0"
+```
+
+The [REUSE Specification](/spec) documents more thoroughly how `REUSE.toml` can
+be used.
+
+## ... override erroneous REUSE information? {#override-info}
 
-If you have a directory containing many files, it may not be easy or practical
-to edit every file to contain a header. While doing this would be ideal, there
-is an alternative. By creating the file `.reuse/dep5` in the root of your
-project, you can bulk-license a directory. Example:
+You may have a file that contains copyright or licensing tags that are wrong.
+This may be on purpose, or maybe you have good reasons not to edit these files.
+You have a few options to instruct REUSE to ignore the erroneous information and
+to use correct information instead.
 
+1. Use [the `REUSE-IgnoreStart` and `REUSE-IgnoreEnd` tags](#exclude-lines)
+   inside of the file.
+2. Use [a `.license` file to override the contents of the
+   file](#uncommentable-file).
+3. Use [a `REUSE.toml` file](#bulk-license) with the `precedence = "override"`
+   key-value.
+
+<!-- REUSE-IgnoreStart -->
+
+## ... add additional REUSE information to a file without editing the header? {#aggregate-info}
+
+There is a specific corner case where a file may contain valid REUSE information
+(usually copyright holders), but it is not desirable for you to edit this file's
+comment header by hand, even though there is more information that should apply
+to this file. A good example of this corner case is `.po` files provided by
+translators. These may or may not contain copyright and licensing information,
+depending on the translator's translation software and settings.
+
+You can add additional information using the `precedence = "aggregate"`
+key-value in your [`REUSE.toml` file](#bulk-license). An example looks like
+this:
+
+```toml
+[[annotations]]
+path = "po/*.po"
+precedence = "aggregate"
+SPDX-FileCopyrightText = "Contributors to My Project"
+SPDX-License-Identifier = "GPL-3.0-or-later"
 ```
-Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
-Upstream-Name: my-project
-Upstream-Contact: Jane Doe <jane@example.com>
-Source: https://git.example.com/jane/my-project
-
-Files: resources/img/*
-Copyright: 2017 Jane Doe <jane@example.com>
-License: CC-BY-4.0
-
-Files: resources/vid/*
-Copyright: 2017 Jane Doe <jane@example.com>
-           2017 John Doe <john@example.com>
-License: CC0-1.0
+
+In the above example, 'Contributors to My Project' is added to the file's
+copyright holders (if any), and 'GPL-3.0-or-later' is added to the file's
+licenses (if any).
+
+## ... copy someone else's work? {#copy-work}
+
+If someone else has made their work available for you to use and copy, you can
+incorporate their work into your project. When you put the work in one of the
+files in your project, you should add an `SPDX-FileCopyrightText` tag for the copyright
+holder(s) and an `SPDX-License-Identifier` tag for the license(s) under which
+the work was made available.
```

Comment:
```suggestion
the work was made available. Unless, of course, the author wishes to remain unattributed.
```

Reply:
This caveat doesn't really help the reader, I think. If we want to add advice on how to deal with anonymous copyright holders, it should be a separate question. Would you be willing to write that?

Reply:
@carmenbianca , you mean a general section about posting _your own_ stuff pseudo-/anonymously or one that deals with also third party?

The issue I’d like to avoid _here_ is that people don’t think they need to go on a wild goose hunt to figure out who the author was of every little thing, if the authors themselves didn’t care signing their own work.

Reply:
I meant third-party stuff without names attached.

I think I understand better now why you want to add this, but:

- The proposed addition is a very roundabout way of saying that.
- I'm not sure it's a sound recommendation. Obviously, I don't hunt down every copyright holder either and usually make a best guess when it's especially arduous to figure out, but I don't feel at ease recommending that practice to other people, because I know it's not strictly The Best Thing To Do.

I'm convinced that there may be a way to instruct the user to not sweat it too much when the author didn't do their job of attributing themselves very well, but it needs a little more finesse.

Reply:
There’s two use cases here I’d like to avoid:

- being more Catholic then the Pope – if the author didn’t care to make use of their rights, it’s not your obligation to correct that
- privacy breach – when the author very consciously did not want to include their name for whatever reason (e.g. _really_ does not care, safety reasons like dead-name or NDA issues)

Reply:
That said, I am leaning towards agreeing with you not to complicate things in this section. I’ll suggest something in the following question/answer, as it may fit there better.